crates/matrix-sdk-ui/src/timeline/event_item/content/reply.rs

```diff
@@ -68,6 +68,12 @@ pub struct EmbeddedEvent {
     pub sender: OwnedUserId,
     /// The profile of the sender of the related embedded event.
     pub sender_profile: TimelineDetails<Profile>,
+    /// The timestamp of the event.
+    pub timestamp: MilliSecondsSinceUnixEpoch,
+    /// The unique identifier of this event. This is the transaction ID
+    /// for a local echo that has not been sent and the event ID for a
+    /// local echo that has been sent or a remote event.
```

Comment:
nit: let's use the shorthand format for getting a short description with LSP:

```suggestion
    /// The unique identifier of this event.
    ///
    /// This is the transaction ID for a local echo that has not been sent and the event ID for a
    /// local echo that has been sent or a remote event.
```